steps/mentor-calibre-lvs/lvs.runset.template

```diff
@@ -27,5 +27,5 @@
 *cmnTranscriptEchoToFile: 1
 *cmnRunMT: 1
 *cmnRunHyper: 1
-*cmnV2LVS_SPICEIncFiles: inputs/adk/*.cdl inputs/*.spi inputs/*.sp
+*cmnV2LVS_SPICEIncFiles: ${lvs_adk}/*.cdl inputs/*.spi inputs/*.sp
```

Comment:
What you actually want is to supply _another_ adk right? One with multi VTs?

So one proposal is:

1. Add an additional parameter for ${lvs_extra_spice_include} to this line here.
2. Attach the 2nd adk as an input
3. Graph should set the parameter "lvs_extra_spice_include" to the 2nd adk's *.cdl

Reply:
So something like this...? In adks/tsmc16/configure.yml (tsmc16 repo) I add new outputs and such something like
```
outputs:
  - adk_lvs2

commands:
  - 'if [ -z "{adk_lvs2_view}" ];
     then ln -sf /dev/null adk_lvs2;
     else ln -sf ../{adk_lvs2_view} adk_lvs2;
     fi'

parameters:
  adk_lvs2_view: ""
```

Then in full_chip/construct.py (garnet repo) I add new parameters
```
  parameters = {
    # LVS
    'adk_lvs2_view' : 'multivt',
    'lvs_extra_spice_include'   : 'inputs/adk_lvs2/*.cdl',
```

In the lvs step configure.yml (mflowgen repo) I add new inputs and parameters
```
inputs:
  - adk
  - adk_lvs2

parameters:
  lvs_extra_spice_include : ""
```

..and then finally in the lvs template (mflowgen repo) I do this:
```
- *cmnV2LVS_SPICEIncFiles: inputs/adk/*.cdl inputs/*.spi inputs/*.sp
+ *cmnV2LVS_SPICEIncFiles: inputs/adk/*.cdl ${lvs_extra_spice_include} inputs/*.spi inputs/*.sp
```